sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -158,7 +158,7 @@ object Size {
        4
   """,
   since = "3.3.0",
-  group = "collection_funcs")
```

Comment:
We should actually replace all of `collection_funcs` case: https://github.com/apache/spark/blob/af1dba7ca501fd9372b158793119163e3fcd1f24/sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionDescription.java#L110-L116

Reply:
For others, maybe use `agg_funcs` (?).

Reply:
Should probably pick something closer for others for `Size` or add a new group if they do not fit.

Reply:
https://github.com/apache/spark/pull/30867 

<img width="875" alt="image" src="https://user-images.githubusercontent.com/1475305/207274575-ae5bac57-fda0-45bc-9a26-baf59c08ae3a.png">


From SPARK-33124, I think `Size` should use `collection_funcs`, it is similar to `Reverse` and `ElementAt`

Reply:
Hm, I see. Okay, makes sense to me